.github/pull_request_template.md

```diff
@@ -6,8 +6,8 @@
 
 - [ ] I have added tests to support my implementation
 - [ ] I have read and agree with
-  the [contributing documentation](https://github.com/Shopify/checkout-kit-android/blob/main/.github/CONTRIBUTING.md)
+  the [contributing documentation](https://github.com/Shopify/checkout-sheet-kit-android/blob/main/.github/CONTRIBUTING.md)
 - [ ] I have read and agree with
-  the [code of conduct documentation](https://github.com/Shopify/checkout-kit-android/blob/main/.github/CODE_OF_CONDUCT.md)
+  the [code of conduct documentation](https://github.com/Shopify/checkout-sheet-kit-android/blob/main/.github/CODE_OF_CONDUCT.md)
 - [ ] I have updated any documentation related to these changes.
-- [ ] I have updated the [README](https://github.com/Shopify/checkout-kit-android/blob/main/README.md) (if applicable).
+- [ ] I have updated the [README](https://github.com/Shopify/checkout-kit-sheet-android/blob/main/README.md) (if applicable).
```

Comment:
This URL should be: `https://github.com/Shopify/checkout-sheet-kit-android/blob/main/README.md`